.github/workflows/deploy.yml

```diff
@@ -40,23 +54,29 @@ jobs:
 
       - uses: actions/checkout@v2
 
+      - uses: docker/setup-qemu-action@v1
+        name: Set up QEMU
+        if: ${{ matrix.cibw_archs == 'aarch64' || matrix.cibw_archs == 'ppc64le' }}
+
       - uses: actions/setup-python@v2
         with:
           python-version: '3.7'
 
       - name: Install cibuildwheel
         run: |
           python -m pip install --upgrade pip
-          python -m pip install cibuildwheel==1.5.5
+          python -m pip install cibuildwheel==2.1.1
 
       - name: Build wheels
         run: |
-          python -m cibuildwheel --output-dir wheelhouse
+          python -m cibuildwheel --output-dir dist
```

Comment:
The fact we were using `wheelhouse` before as the directory but later on looking for `dist`, is this why things weren't working?

Reply:
I think one of the main reasons, as the pypa CI step expects a `dist` directory.
The artifacts are also stored as `zip.zip`, which may or not not be supported by the upload to PyPI, so I added a separate steps to explicitly extract them before any attempt made.